models/foundation/ethereum/foundation_ethereum_sources.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+
+sources: 
+  - name: foundation_ethereum
+    freshness:
+      warn_after: { count: 12, period: hour }
+      error_after: { count: 24, period: hour }
```

Comment:
could you remove the `error_after` line here ? We're moving away from errors thrown because of source freshness (but keeping warnings)